src/Microsoft.DotNet.Interactive/KernelSupportsNugetExtensions.cs

```diff
@@ -189,25 +192,16 @@ internal static KernelCommandInvocation DoNugetRestore()
                         return;
                     }
 
-                    var messages = new Dictionary<PackageReference, string>(new PackageReferenceComparer());
-                    var displayedValues = new Dictionary<string, DisplayedValue>();
+                    InstallPackagesMessage.RegisterInstallPackagesMimeType(PlainTextFormatter.MimeType);
```

Comment:
We shouldn't do this on every call, because it might overwrite people's custom formatters.

You can add these registrations to `DefaultPlainTextFormatterSet` and `DefaultHtmlFormatterSet`.